frontend/cypress/integration/featureFiles/wizard_request_routing.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Service Details Wizard: Request Routing
+
+  User navigates to the service details page and open the Kiali Wizard to generate a Request Routing scenario.
+
+  Background:
+    Given user is at administrator perspective
+    And user opens the namespace "bookinfo" and "reviews" service details page
+
+  @wizard-request-routing
+  Scenario: Create a Request Routing scenario
+    When user clicks in the "Request Routing" actions
+    And user sees the "Create Request Routing" wizard
+    And user clicks in the "Request Matching" tab
+    And user clicks in the "headers" request matching dropdown
+    And user types "end-user" in the matching header input
+    And user clicks in the "exact" match value dropdown
+    And user types "jason" in the match value input
+    And user adds a match
+    And user clicks in the "Route To" tab
+    And user types "100" traffic weight in the "reviews-v2" workload
+    And user adds a route
+    And user clicks in the "Request Matching" tab
+    And user clicks in "headers [end-user] exact jason" matching selected
+    And user clicks in the "Route To" tab
+    And user types "100" traffic weight in the "reviews-v3" workload
+    And user adds a route
+    And user previews the configuration
+    And user creates the configuration
+    Then user sees a table with headings
+      | Name | Status |
+    And table length should be 2
```

Comment:
is it supposed to verify the content of created DR and VS? 
also to make sure that correct traffic weights are set

Reply:
+1

Perhaps I can check that in a new scenario (in that file), before deleting the generated Istio Config.

I'll update the test.

Reply:
I've updated the scenarios, not 100% completed but it expands to navigate to the details and perform some checks in the references and the lines.

P.S.: Parsing the ACE editor is "interesting" :D

Reply:
I don't think is good to combine REST API calls (or system calls like the kubectl) in the UI test.

If that API is private, I think that the UI test should be solid and support that the private API can change without affecting the test.

But in the same way we have introduced the "data-test" attribute, I think we can add a helper in the UI that would simplify to check and parse the resulting object.

Reply:
I've addressed this within the dom, the "text()" invoke in an inner content "div" used by the ACE editor works reasonably well to make assertions (and no need to add extra overhead to that component).